DockerENT/docker_plugins/docker_process_info.py

```diff
@@ -0,0 +1,56 @@
+"""Docker running processes-info scan plugin."""
+from DockerENT.utils import utils
+
+import docker
+import logging
+
+_log = logging.getLogger(__name__)
+
+_plugin_name_ = 'procinfo'
+
+
+def scan(container, output_queue):
+    """Docker running processes-info plugin scan.
+
+    :param container: container instance.
+    :type container: docker.models.containers.Container
+
+    :param output_queue: Output holder for this plugin.
+    :type output_queue: multiprocessing.managers.AutoProxy[Queue]
+
+    :return: This plugin returns the object in this form.
+    {
+        _plugin_name_: {
+            'test_performed': {
+                'results':  []
+            }
+        }
+    }
+    """
+    res = {}
+
+    _log.info('Staring {} Plugin ...'.format(_plugin_name_))
+
+    api_client = docker.APIClient()
+    container_obj = api_client.containers(container.short_id)
+    top_result = api_client.top(container_obj[0])
+
+    running_processes = {
+        "Running_Processes": {
+            "msg": " Check all running processes inside container. ",
```

Comment:
Keeping a plugin to perform cmd check is not a great idea, Just a suggestion can you keep this plugin to pull maximum process-related information?
- Like which processes are mapped to host system.
- Which are privileged.
- If those processes are taking input from n/w. etc.

Reply:
@nishitm You can also check which process-related CIS benchmarks are can be included in this plugin.